plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -189,5 +192,68 @@ public void testFilesTable()
                         "('split_offsets', 'array(bigint)', '', '')," +
                         "('equality_ids', 'array(integer)', '', '')");
         assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$files\"");
+
+        MaterializedRow filesResult = computeActual("SELECT file_format, record_count, " +
+                "lower_bounds, upper_bounds, file_path FROM test_schema.\"test_table$files\" order by file_path").getMaterializedRows().get(0);
+
+        // file format
+        assertEquals(filesResult.getField(0), "ORC");
```

Comment:
I'd just compare all rows  (do not necessarily need order by) . See some examples in BaseIcebergConnectorTest

Reply:
will check

Reply:
@phd3 here intent was to add a bit more coverage to the `files` test as well.
As it was not comparing the results of the `$files`, so added the `lower_bound` and `upper_bound` as well.

are you referring to this test: https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java#L2416-L2427

something like this will compare the file_size and count?

```
        MaterializedResult files = computeActual("SELECT file_path, record_count, file_size_in_bytes FROM \"test_file_size_in_manifest$files\"");
        long totalRecordCount = 0;
        for (MaterializedRow row : files.getMaterializedRows()) {
            String path = (String) row.getField(0);
            Long recordCount = (Long) row.getField(1);
            Long fileSizeInBytes = (Long) row.getField(2);

            totalRecordCount += recordCount;
            assertThat(fileSizeInBytes).isEqualTo(Files.size(Paths.get(path)));
        }
        // Verify sum(record_count) to make sure we have all the files.
        assertThat(totalRecordCount).isEqualTo(2);
```

Reply:
@osscm We don't need to validate the correctness of the table. I was talking about comparing results like the following in BICT

```
        MaterializedResult result = computeActual("SHOW STATS FOR " + tableName);
        MaterializedResult expectedStatistics =
                resultBuilder(getSession(), VARCHAR, DOUBLE, DOUBLE, DOUBLE, DOUBLE, VARCHAR, VARCHAR)
                        .row("col", null, null, 0.0, null, "-10.0", "100.0")
                        .row(null, null, null, null, 2.0, null, null)
                        .build();
        assertEquals(result, expectedStatistics);
```

Reply:
done the changes @phd3